scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -547,7 +547,7 @@ public boolean restartTaskOnError(final JobId jobId, final String taskName)
             Boolean result = infrastructure.getClientOperationsThreadPool().submit(new Callable<Boolean>() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)